packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/ScatterPlotViewer/components/ScatterPlot/components/Selections/DeleteButton.js

```diff
@@ -0,0 +1,92 @@
+import PropTypes from 'prop-types'
+import styled, { css } from 'styled-components'
+
+const StyledGroup = styled('g')`
+  &:focus {
+    ${props => css`outline: solid 4px ${props.focusColor};`}
+  }
+
+  &:hover {
+    cursor: pointer;
+  }
+`
+const DEFAULT_HANDLER = () => true
+  
+export default function DeleteButton({
+  colors,
+  cx,
+  cy,
+  fill = 'black',
+  label,
+  onDelete = DEFAULT_HANDLER,
+  opacity = 1,
+  rotate = 0,
+  stroke = 'white',
+  strokeWidth = 1.5
+}) {
+  const focusColor = colors.focus
+  const RADIUS = (screen.width < 900) ? 11 : 8
+  const CROSS_PATH = `
+    M ${-1 * RADIUS * 0.7} 0
+    L ${RADIUS * 0.7} 0
+    M 0 ${-1 * RADIUS * 0.7}
+    L 0 ${RADIUS * 0.7}
+  `
+  const transform = `
+    translate(${cx}, ${cy})
+    rotate(${rotate})
+  `
+  function onKeyDown(event) {
+    switch (event.key) {
+      case 'Enter':
+      case ' ': {
+        return onPointerDown(event)
+      }
+      default: {
+        return true
+      }
+    }
+  }
+
+  function onPointerUp(event) {
+    event.preventDefault()
+    event.stopPropagation()
+    onDelete()
+    return false
+  }
+
+  return (
+    <StyledGroup
+      aria-label={label}
+      focusable
+      focusColor={focusColor}
+      onKeyDown={onKeyDown}
+      onPointerUp={onPointerUp}
+      opacity={opacity}
+      role='button'
+      stroke={stroke}
+      strokeWidth={strokeWidth}
+      tabIndex='-1'
```

Comment:
Nesting one focusable element inside another focusable element isn’t advisable and should generate an accessibility error in the storybook. 

Imagine, for example, a button nested inside a link (or another button.) What would the expected behaviour be on click? (Those are both invalid HTML, by the way.)

Reply:
Most of the code here is copied from the drawing tools. I’ll take another look at it.

Reply:
Wait, these aren’t nested inside selections. Maybe they should be in the keyboard tab order. It might be irritating to have to tab through the chart twice, though.

Reply:
Here’s where that tabindex was originally changed to -1. 
https://github.com/zooniverse/front-end-monorepo/commit/ae33b76adb2fe69742cf3ce1638f6aa646e28f77